src/pal/src/include/pal/context.h

```diff
@@ -468,6 +472,19 @@ inline static void CONTEXTSetPC(LPCONTEXT pContext, DWORD64 pc)
 #endif
 }
 
+inline static DWORD64 CONTEXTGetFP(LPCONTEXT pContext)
```

Comment:
I understand, but we already have GetPC and GetSP, so naming it e.g. GetFramePointer would look a bit out of line.